flake.nix

```diff
@@ -21,7 +21,17 @@
     forAllSystems = nixpkgs.lib.genAttrs ["x86_64-linux" "aarch64-linux"];
   in {
     nixosModules = {
-      hjem = ./modules/nixos;
+      hjem = {
+        imports = [
+          ./modules/nixos
+          self.nixosModules.hjem-lib
```

Comment:
Very silly nit (and therefore feel free to ignore) but self.nixosModules should come before the path since the former is a prerequisite for the latter.

\- NotASandro